### PR TITLE
Snipamupdates

### PR DESCRIPTION
**Pull request by bhillkeyfactor**

### CODE REVIEW

**Comment by bhillkeyfactor** [comment]
Citrix Adc, Alias fixes, SNI Feature, Remove Nitro Source and Pam Support

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+1.0.3
```
Comment (fiddlermikey) [inline CHANGELOG.md:1]: I do not believe we ever released a 1.0.3 and these updates appear to be for 2.0.0 and this should be updated.

File: RunTest.bat
Comment (fiddlermikey) [inline RunTest.bat:1]: Should this file be removed? Is it an example? It contains a local user account for the first cd command

File: integration-manifest.json
Comment (fiddlermikey) [inline integration-manifest.json:1]: Universal Orchestrators now require kfutil definitions for cert store types in the manifest

**Review by fiddlermikey — CHANGES_REQUESTED** [review]
Please review and update the changelog and manfiest. Remove the runtest.bat file

**Pull request merged**